tests/curves/p256_tests.rs

```diff
@@ -0,0 +1,137 @@
+// tests/curves/p256_tests.rs
```

Comment:
sorry if obvious, but what's the relationship between this and the reasoner?

Reply:
ah good catch - I was running goose from this dir and having it write some rust code to test out o1.